modules/nux/modules.php

```diff
@@ -80,6 +81,9 @@ public function process() {
         if (data_source_available($modules, 'smtp')) {
             $smtp_servers = count(Hm_SMTP_List::dump(false));
         }
+        if (data_source_available($modules, 'jmap')) {
+            $jmap_servers = count(Hm_IMAP_List::dump(false));
```

Comment:
Here you will get imap servers to, i mean it will combine imap ad jmap. If soo then we don't need to add an other link for jmap.

Reply:
With the first idea it was already well started, it was rather necessary to apply the filter here: https://github.com/cypht-org/cypht/blob/master/modules/nux/modules.php#L64 without forgetting in this case to do it for this too: https://github.com/cypht-org/cypht/blob/master/modules/nux/modules.php#L75